scripts/manage-identity.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Parts of this script re-implement logic from the rust code, could this be an additional `bin` inside the proxy codebase?

Reply:
Alternatively as the rest of the id management lives inside the native app shell, it could be moved there.

Reply:
This is just an intermediate script, which we also want to provide to user who are not building from source. In the long term, it should be replaced with an in-app interface to select your current identity.